cg/store/models.py

```diff
@@ -815,6 +815,14 @@ class Sample(Base, PriorityMixin):
         back_populates="sample", cascade="all, delete"
     )
 
+    def __init__(self, **kwargs):
```

Comment:
why this?

Reply:
Because otherwise it crashed when I passed keyword arguments to it which does not exist in the database model. So concentration and stuff which we only have in LIMS